src/main/java/database/DatabaseHandler.java

```diff
@@ -31,8 +27,10 @@
      */
     List<Notification> getUnreadNotifications(int userId);
 
+
     /**
-     * Close a connection with the database
+     * Mark all notifications of a user as read
+     * @param userId: The owner of the notifications
      */
-    void disconnect();
+    void markAllNotificationsAsRead(int userId);
```

Comment:
I think it will be better if this method return a boolean that indicates that the update succeeds or fails.

Reply:
If an update fails, it will throw an `ArangoDBException`, I think it is better to have an exception handling and logging mechanism in the messages handler instead of passing boolean values around. What do you think?

Reply:
Yes I agree but I mean that any `ArangoDBException` should be handled in ArangoHandler and boolean indicates if the operation fails in order to output an error response to the issuer of the request. You can instead of using a boolean, wrap `ArangoDBException` into custom exception that will be used to construct response to issuer. Do you find this approach correct?

Reply:
Yes I like wrapping the exception. Let's do this later when we have the message queuing handler.